src/main/java/org/jabref/gui/icon/InternalMaterialDesignIcon.java

```diff
@@ -41,7 +43,7 @@ public Node getGraphicNode() {
         Ikon icon = icons.get(0);
         FontIcon fontIcon = FontIcon.of(icon);
         fontIcon.getStyleClass().add("glyph-icon");
-        color.ifPresent(fontIcon::setIconColor);
+        color.ifPresent(color -> fontIcon.setStyle(fontIcon.getStyle() + String.format("-fx-fill: %s;", ColorUtil.toRGBCode(color))));
```

Comment:
Do you know why `setIconColor` doesn't work? Maybe add a short comment so that we will not revert it in the future by accident.

Reply:
Sorry, I have no idea.

Reply:
It has something to do with the style class. If you uncomment the line "getStyleClass.add("glyph-icon")" than it works properly. However, then the rest of the icons are black because that property is set in the css (and is obviously different for dark and base.css) and therefore overrides the color of the 'FontIcon". 

So add a comment like:
Override the default color from the css files